docker-compose.yml

```diff
@@ -1,20 +1,18 @@
-version: '3.2'
+version: "3.2"
 services:
   postgres:
     image: postgres:latest
```

Comment:
This is more of a note/question and I won't hold this up as it hasn't changed. I thought I would try to setup a local copy of the db so I rsync'd the `pgdata` directory from linode to my machine. When I try to start the server the postgres container won't start as it says there is a mismatch between the version of postgres running and the data that is in the directory.

Should we be pinning the major version and not using latest for reproducibility?